setup.py

```diff
@@ -136,32 +136,8 @@ def build_extension(self, ext):
     )
     ext_modules = []
 
-
```

Comment:
You were able to get rid of all this because of the way that requirements.txt is handled? That's awesome!